README.md

```diff
@@ -151,7 +151,8 @@ In order to check how your Apache Kafka cluster is behaving, the Canary provides
 | `consumer_timeout_join_group_total`     | The total number of consumers not joining the group within the timeout            |
 | `consumer_refresh_metadata_error_total` | Total number of errors while refreshing consumer metadata                         |
 | `records_consumed_latency`              | Records end-to-end latency in milliseconds                                        |
-| `connection_error_total`                | Total number of errors while checking the connection to Kafka brokers             |
+| `connection_error_total`                | *Deprecated - use connection_total* Total number of errors while checking the connection to Kafka brokers|
+| `connection_total`                      | Total of outcomes while checking the connection to Kafka brokers                  |
```

Comment:
```suggestion
| `connection_total`                      | Total number of connections to Kafka brokers                  |
```

Reply:
I wasn't sure what _outcomes_ meant here. Suggestion assumes we're just returning total number of connections.  We could add succeeded or failed too: _Total number of (failed or successful) connections to Kafka brokers_